src/features/resources/ResourceItem.tsx

```diff
@@ -53,32 +75,117 @@ const ResourceItem = ({ resource, event }: { resource: Types.Resource; event: an
     updateFavorites();
   };
 
+  const { data, isLoading, isSuccess } = useStatus();
+  const [showDialog, setShowDialog] = useState(false);
+  const [itSystemError, setItSystemError] = useState(false);
+  const [systemCheckedAt, setSystemCheckedAt] = useState(new Date());
+  const cancelRef = useRef<HTMLButtonElement>(null);
+  const open = () => setShowDialog(true);
+  const close = () => setShowDialog(false);
+
+  useEffect(() => {
+    if (isSuccess) {
+      // look for the resource's corresponding IT system
+      if (resource.hasOwnProperty('itSystem') && data !== undefined) {
```

Comment:
rather than `data !== undefined` you can check its truthiness with just the double amphersand.. 

so something like `if (resource.hasOwnProperty('itSystem') && data) {` should work.. check the same situation a few lines lower